docs/reference/tags/form_tags/create_line_item/create_line_item.md

```diff
@@ -7,8 +7,9 @@ grand_parent: Reference
 
 # Create line item
 
-The `create_line_item` tag renders a form which acts as a wrapper for a single item, an item can be a [variant]({% link docs/reference/objects/product/variant/index.md %}) or an [extra]({% link docs/reference/objects/product/extra.md %}).
-Extra HTML input tags can be used to add item quantity, modifiers, adult count or start and end date.
+The `create_line_item` tag renders a form which acts as a wrapper for a single item, an item can be a [variant]({% link docs/reference/objects/product/variant/index.md %}) or an [extra]({% link docs/reference/objects/product/extra.md %}).<br>
```

Comment:
pet peeve nit: the break tag is meant for addresses & poems, you can add a blank line to start a new paragraph!

Reply:
Adding a blank line is literally adding a blank line and not a paragraph 😞